app/src/main/java/com/sdp/movemeet/utility/ImageHandler.java

```diff
@@ -84,4 +89,17 @@ public void onFailure(@NonNull Exception e) {
         });
     }
 
+    /**
+     * Convert the Firebase Storage URL of the image into a path in Firebase Storage
+     *
+     * @param imageUrl URL of the image in Firebase Storage
+     * @return the converted image path
+     */
+    public static String convertURLtoPath(String imageUrl) {
+        int startIdx = imageUrl.indexOf(ChatActivity.CHATS_CHILD);
+        int endIdx = imageUrl.indexOf(ImageHandler.CHAT_IMAGE_NAME) + ImageHandler.CHAT_IMAGE_NAME.length();
+        String imagePath = imageUrl.substring(startIdx, endIdx);
+        imagePath = imagePath.replace("%2F", ImageHandler.PATH_SEPARATOR);
```

Comment:
Why do we need this replace ?

Reply:
I basically extracted this method from MessageViewHolder in order to test it with a Unit test.